pgpainless_signer/src/test/java/org/eclipse/packager/rpm/signature/pgpainless/AbstractSigningTest.java

```diff
@@ -0,0 +1,141 @@
+package org.eclipse.packager.rpm.signature.pgpainless;
+
+import org.bouncycastle.bcpg.ArmoredInputStream;
+import org.bouncycastle.bcpg.BCPGInputStream;
+import org.bouncycastle.openpgp.PGPObjectFactory;
+import org.bouncycastle.openpgp.PGPSecretKeyRing;
+import org.bouncycastle.openpgp.bc.BcPGPObjectFactory;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public class AbstractSigningTest {
+
+    public static final String KEY = "-----BEGIN PGP PRIVATE KEY BLOCK-----\n" +
```

Comment:
This is probably better as a file in test/resources

Reply:
Yeah, I just copied it from the rpm test resources. I'll see if it is possible to reuse the existing test key somehow.